src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contacts/edit/details/main.js

```diff
@@ -461,17 +475,9 @@ define([
             // listen for change after TAGS and BIRTHDAY-field have been set
             this.sandbox.data.when(this.dfdAllFieldsInitialized).then(function() {
 
-                this.sandbox.dom.on('#contact-form', 'change', function() {
-                    this.sandbox.emit('sulu.tab.dirty');
-                }.bind(this), '.changeListener select, ' +
-                '.changeListener input, ' +
-                '.changeListener textarea');
-
-                this.sandbox.dom.on('#contact-form', 'keyup', function() {
+                this.sandbox.dom.on('#contact-form', 'change keyup', function() {
                     this.sandbox.emit('sulu.tab.dirty');
-                }.bind(this), '.changeListener select, ' +
-                '.changeListener input, ' +
-                '.changeListener textarea');
+                }.bind(this), 'select, ' + 'input, ' + 'textarea');
```

Comment:
@marcelmoosbrugger is there a reason why you don't use "'select, input, textarea'"?